apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -71,6 +73,9 @@ function updateEditor() {
 			// Disable href navigation
 			e.preventDefault();
 			await dispatch( 'core/editor' ).savePost();
+
+			recordTracksEvent( 'calypso_editor_launch_click', {} );
```

Comment:
You can also leave `{}` out and it'll default to it: https://github.com/Automattic/wp-calypso/blob/07f9bdfc34da00a78de94af416e2f0eafa3b757f/packages/calypso-analytics/src/tracks.ts#L168

Not a biggie, up to you!